src/backend/mailbox_manager/migrations/0019_fill_maildomain_support_email.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+
+def fill_support_email(apps, schema_editor):
+    MailDomain = apps.get_model('mailbox_manager', 'MailDomain')
+    for domain in MailDomain.objects.filter(support_email__isnull=True):
+        # FIXME
+        domain.support_email = f"support@{domain.name}"
```

Comment:
I would use a more specific fake email if it needs to be fixed.

Reply:
Christophe just created support-regie@numerique.gouv.fr